.travis.yml

```diff
@@ -11,10 +11,10 @@ branches:
   - master
 
 cache:
-  pip: true
   yarn: true
   directories:
     - node_modules
+    - $HOME/virtualenv/python2.7.13
```

Comment:
This can't be calculated at runtime, can it?

Say, something like `$HOME/virtualenv/python$(python -c 'import platform; print(platform.python_version())')`

That way we don't hit the issue where python version changes and we're no longer caching things.

Reply:
That would work - good idea!